packages/schema/src/schema.graphql

```diff
@@ -67,8 +65,9 @@ type FooterNavigation implements Navigation @menu(menu_id: "footer") {
   items: [NavigationItem]! @lang @resolveMenuItems
 }
 
-interface Page implements ContentHubResultItem {
+interface Page implements ContentHubResultItem @value(json: "\"page\"") {
```

Comment:
What's the reason for `@value` here?

Reply:
Drupal needs a value to resolve the `Page` interface. Even though there is only one subtype within Drupal. I changed it to use the entity bundle. Thats more clear and practical (I hope).